apps/docs/content/guides/guides/components.mdx

```diff
@@ -0,0 +1,170 @@
+
+- 🥇 [Behavior modifications](#customization-methods-details--behavior-modifications): You can use ref, callbacks and controlled properties to modify its behavior.
+
+- 🥈 [Copying component](#customization-methods-details--copying-component): Copy the component code inside your codebase and modify it.
+
+- 🥈 [Custom components using base components](#customization-methods-details--custom-components-using-base-components): Create a new component on top of Design System base components or React Aria Components.
+
+- 🥉 [Custom components using hooks](#customization-methods-details--custom-components-using-hooks): Create a new component on top of Design System hooks or React Aria hooks.
+
+- 🛠️ [Building components from scratch](#customization-methods-details--building-components-from-scratch): Create a component from scratch and leveraging Design Tokens if possible
+
+## Customization Methods Details
+
+These customization options provide product teams with the flexibility to adapt Hopper components to specific requirements, without sacrificing consistency. By using this range of customization methods, teams can meet product goals efficiently while maintaining the benefits of a unified design system. These customization methods are presented in the order you should consider when evaluating your options, with details provided in the sections below.
+
+### 🥇 Direct styling adjustments
+
+For straightforward customization needs, you can make adjustments directly by passing style-related props, adding CSS classes, or setting up refs for access to component elements. This approach allows you to override specific styles while keeping the component's behavior and overall styling intact.
+
+<Expand title="Details">
+#### Styled System Props
+
+Adjust properties such as `backgroundColor`, `width`, or `padding` directly on the component.
+
+```tsx
+<Button backgroundColor="primary" UNSAFE_width="148px" paddingTop="stack-xs">Click me</Button>
+```
+
+#### CSS Classes
+
+Append custom CSS classes to modify styles.
+
+```tsx
+<Button className="custom-class">Click me</Button>
+```
+</Expand>
+
+<Expand title="Example: Creating an AI Card">
+```tsx
+export function AICard({ children, className, ...rest }: CardProps) {
+  const classNames = clsx(className, "ai-gradiant-border");
+   return (
+       <Card className={classNames} {...rest}>
+           {children}
+       </Card>
+   );
+}
+```
+</Expand>
+
+### 🥇 Behavior modifications
+
+To customize a component's behavior, use refs, callbacks and controlled properties. This approach is useful for interactive features and for modifying default component behavior. This approach is ideal when you need to adjust how a component responds to user interactions without altering its visual style or structure. Read more about Controlled vs Uncontrolled components [here](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components).
```

Comment:
Controlled and Uncontrolled are capitalized here. should they be emphasized in another way? Bolded perhaps.